controller/controller.py

```diff
@@ -29,8 +38,24 @@ def receive_from_twin(self):
 
         pass
 
-    def receive_from_rover(self):
+    def receive_from_rover(self, is_online:bool = False):
         """
         Function to digest rover input
+        :param is_online: boolean value to indicate if the rover is live or not
         """
```

Comment:
Would be good to discuss code documentation style. I think it would be good to have more pydoc like this in other parts of the repo.